app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Tag, Tooltip, Avatar, Dropdown } from "neetoui";
+import { Header } from "neetoui/layouts";
+
+import { USER_AVATAR_URL } from "../constants";
+import { dateTimeFormatter, relativeCreationTime } from "../utils";
+
+const { Menu, MenuItem } = Dropdown;
+const Cards = ({ item }) => (
+  <div className="m-3 w-full  border border-gray-200 p-4 shadow-sm  dark:border-gray-500 dark:bg-gray-600">
+    <Header
+      title={item.title}
+      actionBlock={
+        <Dropdown buttonStyle="text" icon={MenuVertical}>
+          <Menu>
+            <MenuItem.Button>Edit</MenuItem.Button>
+            <MenuItem.Button>Delete</MenuItem.Button>
+          </Menu>
+        </Dropdown>
+      }
+    />
+    <div className="mb-4">{`"${item.description}"`}</div>
```

Comment:
Go through prev review comments in this PR.

Reply:
Could you please further clarify this for me? Is the issue with this part of the code or is it just suggesting I have walk through the other PR comments once more?

Reply:
>suggesting I have walk through the other PR review comments once more

@KrishnapriyaSkk Review comments in this PR. Refer: https://github.com/KrishnapriyaSkk/neeto-ui-challenge/pull/8#discussion_r977360905

Reply:
Also read:

https://academy.bigbinary.com/learn-react/pages/semantic-html-and-accessibility